providers/implementations/kdfs/sskdf.c

```diff
@@ -165,7 +165,7 @@ static int kmac_init(EVP_MAC_CTX *ctx, const unsigned char *custom,
     if (custom == NULL)
         return 1;
 
-    params[0] = OSSL_PARAM_construct_octet_string(OSSL_MAC_PARAM_CUSTOM,
+    params[0] = OSSL_PARAM_construct_octet_string(OSSL_KDF_PARAM_CUSTOM,
```

Comment:
Technically this is a MAC setter.. it just sits inside a KDF.. As long as the name doesnt change I dont care..